packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -557,8 +557,11 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
       subMenuProps: item.subMenuProps,
       submenuIconProps: item.submenuIconProps
     } as IContextualMenuItem;
+
     return React.createElement('button',
-      getNativeProps(itemProps, buttonProperties),
+      assign({}, getNativeProps(itemProps, buttonProperties), {
```

Comment:
I'm guessing they use assign instead of the spread operator for this kind of thing elsewhere in this file?

Reply:
Yes.. they use if also for renderButton. Is there a convention of what is best to use?

Reply:
I prefer spread, but in this case be consistent with the rest of the file